engine/src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -36,6 +36,9 @@ object SgeBackend {
     def script = jobDescriptor.callRootPath.resolve("script.sh")
     def returnCode = jobDescriptor.callRootPath.resolve("rc")
   }
+
+  private lazy val oldGoogleConf = GoogleConfigurationAdapter.gcloudConf.get
+  lazy val gcsConf = Try(GoogleConfiguration(oldGoogleConf.appName, oldGoogleConf.userAuthMode.getOrElse(oldGoogleConf.cromwellAuthMode)))
```

Comment:
Hmm I'm not sure I understand, every backend is supposed to be totally isolated in theory, so if several backends use the same features there is going to be some copy/paste at some point, unless we create ©MetaBackends ?